docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_navigation:
+  title: Share work packages
+  priority: 963
+description: How to share work packages in OpenProject.
+keywords: work package, share, share work packages
+---
+
+# Share work packages
+
+Since the 13.1 it is possible to share work packages with users that are not members of a project or are not yet registered on your instance. In the latter case a user will need to register in order to view the work package.
+
+> **Note**: Sharing work packages with non-member is an Enterprise add-on and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or  [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free community edition is easy and helps support OpenProject.
+
+To share a work package with a project non-member select the detailed view of a work package and click the **Share** button.
+
+![Share button in OpenProject work packages](openproject_user_guide_share_button_wp.png)
+
+A dialogue window will open, showing the list of all users, who this work package has already been shared with. If the work package has not yet been shared, the list will empty. 
+
+> **Note**: For sharing with non members to be available, a respective role needs to be assigned to the **Non member role**. 
+
+![List of users with access to a work package in OpenProject](openproject_user_guide_shared_with_list.png)
+
+If the list contains multiple users you can filter it by Type or Role. 
+
+Following user types are available as filters:
+
+![Filter list of users by user type](openproject_user_guide_sharing_member_type_filter.png)
+
+- Project member - returns all users that are project members
+- Not project member - returns all users that are not project members
+- Project group - returns all users that are members of the group, which includes the project
+- Not project group - returns all users that are not members of the group, which includes the project
+
+Following user roles are available as filters:
+
+![Filter list of users by user role](openproject_user_guide_sharing_member_role_filter.png)
```

Comment:
I don't know if we want to specify the capacities of each role (what the user can do with each of them). Opinions @HDinger?

Reply:
It probably makes sense to explain the capacitites briefly. In English, there are pretty self-explanatory, but in other languages the translations might not as intuitive. (e.g. the german translation for "view" is currently "Ansicht" which even I did not understand directly). 

It might be worth to mention that filters show you only the explicitly granted permissions, not the inherited ones. 
So imagine a user who is shared with permission "view" and the user is part of a group which is shared with as well with permission "comment" . When I filter for "comment", I will only see the group, although the user inherited that permission as well. This is indicated via the status text below the group but it might be worth mentioning nevertheless.